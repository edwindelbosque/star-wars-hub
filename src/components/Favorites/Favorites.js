import React from 'react';
import './Favorites.scss';
import CharactersContainer from '../CharactersContainer/CharactersContainer';

const Favorites = ({ favorites }) => {
  return (
    <section>
      <h1>Favorite</h1>
      <CharactersContainer characters={favorites} />
    </section>
  )
}

export default Favorites;