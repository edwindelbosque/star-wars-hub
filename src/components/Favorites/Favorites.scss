.add-characters {
  margin-top: 80px;
  color:rgb(218, 218, 218);
  font-style: italic;
  font-weight: 300;
  font-size: 40px;
  height: 500px;
  background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(149, 207, 255, 0.733), rgba(0, 0, 0, 0));
}