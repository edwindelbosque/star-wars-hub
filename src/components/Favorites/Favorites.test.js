import React from 'react';
import { shallow } from 'enzyme';
import Favorites from './Favorites';

describe('Favorites', () => {
  it('component should match the snapshot', () => {
    const wrapper = shallow(
      <Favorites favorites={[{ favorite1: 'some name' }]} />
    )
    expect(wrapper).toMatchSnapshot();
  });
})