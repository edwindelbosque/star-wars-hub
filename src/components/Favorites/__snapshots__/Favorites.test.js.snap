// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites component should match the snapshot 1`] = `
<section>
  <h1>
    Favorite Characters
  </h1>
  <CharacterCard
    key="0"
  />
</section>
`;
